assembl/graphql/schema.py

```diff
@@ -662,6 +722,34 @@ def mutate(root, args, context, info):
             for attr, value in kwargs.items():
                 setattr(thematic, attr, value)
             db = thematic.db
+
+            # add uploaded image as an attachment to the idea
+            image = args.get('image')
+            if image is not None:
+                filename = os.path.basename(context.POST[image].filename)
+                mime_type = mimetypes.guess_type(filename)[0]
```

Comment:
You're right I can just use context.POST[image].type. When I coded that, it was only via a test so I didn't know I had this info in real.
```
(Pdb) context.POST[image]
FieldStorage('variables.image', u'9Xr4dPUMDZJG.jpg')
(Pdb) pp context.POST[image].__dict__
{'_FieldStorage__file': None,
 'disposition': 'form-data',
 'disposition_options': {'filename': '9Xr4dPUMDZJG.jpg',
                         'name': 'variables.image'},
 'done': 1,
 'file': <open file '<fdopen>', mode 'w+b' at 0x7f55a763a6f0>,
 'filename': u'9Xr4dPUMDZJG.jpg',
 'fp': <open file '<fdopen>', mode 'w+b' at 0x7f55a763a810>,
 'headers': <rfc822.Message instance at 0x7f55a962f3b0>,
 'innerboundary': '',
 'keep_blank_values': True,
 'length': -1,
 'list': None,
 'name': 'variables.image',
 'outerboundary': '----WebKitFormBoundarySFTYYA9mfH6Q8WEO',
 'qs_on_post': None,
 'strict_parsing': 0,
 'type': 'image/jpeg',
 'type_options': {}}
```